x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/ml/high_sum_total_sales.json

```diff
@@ -0,0 +1,39 @@
+{
+  "groups": ["kibana_sample_data"],
+  "description": "Kibana sample eCommerce data : detect anomalies in total sales",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean total sales",
+        "function": "high_sum",
+	"field_name": "taxful_total_price",
+	"over_field_name": "customer_full_name.keyword"
+      }
+    ],
+    "influencers": ["customer_full_name.keyword", "category.keyword"]
+  },
+  "analysis_limits": {
+    "model_memory_limit": "10mb"
+  },
+  "data_description": {
+    "time_field": "order_date"
+  },
+  "model_plot_config": {
+    "enabled": true
+  },
+  "custom_settings": {
+    "created_by": "ml-module-sample",
+    "custom_urls": [
+      {
+        "url_name": "Raw data",
+        "url_value": "kibana#/discover?_g=(time:(from:\u0027$earliest$\u0027,mode:absolute,to:\u0027$latest$\u0027))&_a=(index:ff959d40-b880-11e8-a6d9-e546fe2bba5f)"
```

Comment:
Did you consider passing the partitioning field to the Discover page? I wonder if that might be more useful than showing data over all customers?

Reply:
Yep, it was more useful with the partitioning field.